pkg/registration/hub/managedcluster/controller.go

```diff
@@ -103,6 +110,15 @@ func (c *managedClusterController) sync(ctx context.Context, syncCtx factory.Syn
 	}
 
 	if !managedCluster.Spec.HubAcceptsClient {
+		// If the ManagedClusterAutoApproval feature is enabled, we automatically accept this cluster and
+		// add an annotation to mark its initial join as accepted.
+		if features.HubMutableFeatureGate.Enabled(ocmfeature.ManagedClusterAutoApproval) {
+			// Only accept this cluster upon its initial join.
```

Comment:
This comment is not needed, the comment above can explain the cases we want to handle.

Reply:
removed